img/Project 3_files/mobile-style.css

```diff
@@ -0,0 +1,167 @@
+html{
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+ }
+/* <!-- Navigation Bar (Menu /Search /ShoppingCar /Login ) :) --> */
+.nav-bar{
+  width: 100%;
+  height: 50px;
+  background-color:rgb(37 211 216 / 98%);
+  border-radius: 5px;
+  display: flex;
+  flex-direction:row;
+  border-style:groove;
+  border-color: rgb(107,236,240) ;
+ 
+}
+.menu-hamburguesa{
+  width: 50px;
+  height: 50px;
+}
+a img{
+  width: 45px;
+  height: 45px;
+  align-items: center;
+}
+.search{
+  display: flex;
+  flex-direction: row;
+  width: 60%;
+  align-items: center;
+}
+.search input{
+  width: 80%;
+  height: 20px;
+  margin-left: 5px;
+}
+.search button{
+  width: 40px;
+  height: 26px;
+  border-radius: 4px;
+  margin-left: 1px;
+  background-color: white;
+}
+.search button img{
+  width: 80%;
+  height: auto;
+}
+.user-container{
+  display: flex;
+  flex-direction: row;
+  width: 30%;
+  height: 50px; 
+}
+.user-container div{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.sign-up{
+  width: 50%;
+  height: 100%;
+  color: aliceblue;
+
+}
+.sign-up img{
+  width: 60%;
+  height: auto;
+  
+}
+.login{
+  width: 50%;
+  height: 100%;
+  color: aliceblue;
+}
+/* <!-- END Navigation Bar (Menu /Search /ShoppingCar /Login ) :) --> */
+
+/* <!-- Hero Image / Video :) --> */
+.hero-home{
+  width: 100%;
+  height: 500px;
+  display: flex;
+  flex-direction: column;
+}
+
+.hero-background{
+  width: 100%;
+  height: 500px;
+
+}
+.hero-background img{
+  width: 400px;
+  height: 400px;
+
+}
+.hero-container{
+  width: 100%;
+  height: 100px;
+  
+}
+.hero-container h1{
+  width: 50%;
+  height: 100px;
+  position: absolute;
+  top:120px;
+  left:68px;
+  text-align: center;
+  font-family: monospace;
+  font-size: 50px;
+  font-weight: bold;
+}
+p{
+  position: absolute;
+  top:415px;
+  font-family: sans-serif;
+  font-size: 25px;
+  text-align: center;
+  color: rgb(233,41,134);
+  text-shadow: black;
+
+}
+.subtitle{
+  font-size: 15px;
+  color: black;
+  font-family: sans-serif;
+  text-align: center;
+  top: 480px;
+  padding: 15px 28px;
+}
+/* <!-- END Hero Image / Video :) --> */
+
+/* <!-- Signup form (Personal data + Interest selection) --> */
+
+.create-account{
+  width: 100%;
+  font-family: sans-serif;
+  color: black;
+  background-image: url(/img/background.png);
+}
+
+fieldset{
```

Comment:
Let's remember we should style by class names, not by selector - needs to be fixed